### ISSUE TITLE
Provide a "collapse all" option in Manage Mods

### ISSUE THREAD

**Issue by itzg** [itzg/minecraft-container-yard#38]
Since the mods/plugins list will likely get very long it would be nice to have an option to collapse the panel-style view into a compact tabular view.

Probably in this spot:
![image](https://cloud.githubusercontent.com/assets/988985/12161543/6f64c69e-b4bc-11e5-8fbf-bfb6ce293765.png)

**Comment by sshipway** [comment]
I agree, being able to collapse down to a single line would be useful.  However, how about allowing each mod to be collapsed/expanded independently?  Maybe a [+] or [-] button on the top left-hand side of the frame, in addition to a 'collapse all' button (and it's sibling, the 'expand all' button)

**Comment by itzg** [comment]
You read my mind. Yes, totally agree.

**Comment by sshipway** [comment]
It would probably be good to do the same on the "Current Containers" list as well; a single line summary (Name, type, port, cmd buttons) with an expansion out to show all the other parameters (mods, world seed/zipfile name, etc etc)

**Comment by itzg** [comment]
:+1:
